Documentation/kvstore.rst

```diff
@@ -4,6 +4,12 @@ Key-Value Store
 Cilium uses an external key-value store to exchange information across multiple
 Cilium instances:
 
+.. note::
```

Comment:
Rather than creating a `.. note::` here, I would suggest creating a new section `Caveats and Limitations` above `Debugging`, and put the text under that section with no highlights.